designs/administrator/4.7/enable-dynamic-plugin-ui/index.md

```diff
@@ -0,0 +1,58 @@
+---
+parent: Administrator
+version: 4.7
+---
+
+# Enabling and disabling dynamic plugin UIs
+
+OpenShift console currently has static plugins that allow teams to contribute features to the UI such as OpenShift Virtualization and OpenShift Container Storage. These plugins live inside console code which means they can only be updated at the same cadence as the console. Newly developed ‘dynamic plugins’ will allow operators to contribute their included UIs at runtime, meaning that operators can update their UIs on their own cadence and more frequently.
+
+Admins installing operators which include dynamic plugins will need to opt-in to having their UIs included in the console UI, which can happen as the operator is being configured for install. Trusted operators could perhaps have this option included by default, whereas other not-explicitly-trusted operators would need to have the user explicitly opt in, as there could be a security risk.
+
+
+Once the operator with the dynamic plugin UI is installed, the user can also turn the dynamic plugin UI on or off in the operator details. The operator list view would also convey any operators with disabled dynamic UIs.
+
+## Enabling dynamic plugin UI during operator install
+
+### Trusted operator install
+
+![Operator trusted install](img/1-1-install-rh.png)
+- A new radio group would appear for operators that include dynamic plugin UIs.
+- Trusted operators would have their dynamic UIs enabled by default.
```

Comment:
What is used to determine if an operator is trusted?

Reply:
> What is used to determine if an operator is trusted?

Operators that come from the `redhat-operators` CatalogSource would have the radio button enabled by default.